superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -29,16 +29,18 @@ import {
   SortIndicator,
   Table,
 } from 'react-virtualized';
-import { getTextDimension } from '@superset-ui/dimension';
+import { createSVGNode, getTextDimension } from '@superset-ui/dimension';
 import { t } from '@superset-ui/translation';
 
 import Button from '../Button';
 import CopyToClipboard from '../CopyToClipboard';
 import ModalTrigger from '../ModalTrigger';
 import TooltipWrapper from '../TooltipWrapper';
 
+const svg = createSVGNode({ container: document.body });
+
 function getTextWidth(text, font = '12px Roboto') {
-  return getTextDimension({ text, style: { font } }).width;
+  return getTextDimension({ text, style: { font }, existingSVGNode: svg }).width;
```

Comment:
Check out my comment here: https://github.com/apache-superset/superset-ui/pull/173/files#r297294252 but I don't think this actually applies the font to the text here.

Reply:
Ah, you're right. I need to pass `font` when creating the SVG node.